lib/Runtime/Language/FunctionCodeGenJitTimeData.cpp

```diff
@@ -812,7 +812,7 @@ namespace Js
 #ifdef FIELD_ACCESS_STATS
         inlineCacheStats(nullptr),
 #endif
-        profiledIterations(GetFunctionBody() ? GetFunctionBody()->GetProfiledIterations() : 0),
+        profiledIterations(GetFunctionBody() && GetFunctionBody()->GetByteCode() ? GetFunctionBody()->GetProfiledIterations() : 0),
```

Comment:
Because the function might have been parsed alright, but the bytecode gen failed